spring-cloud-commons/src/test/java/org/springframework/cloud/commons/util/IdUtilsTests.java

```diff
@@ -117,31 +117,34 @@ public void testUnresolvedServiceId() {
 	@Test
 	public void testServiceIdDefaults() {
 		this.env.setProperty("cachedrandom.application.value", "123abc");
-		then("application:0:123abc").isEqualTo(IdUtils.getResolvedServiceId(this.env));
+		then("application::0:123abc").isEqualTo(IdUtils.getResolvedServiceId(this.env));
```

Comment:
I'm a bit worried about this. This value gets parsed in some places and this would break things.